c/curr/curr_ubi_8.3.sh

```diff
@@ -0,0 +1,80 @@
+#----------------------------------------------------------------------------
+#
+# Package               : curr
+# Version               : 1.0.0
+# Source repo           : https://github.com/otiai10/curr
+# Tested on             : UBI 8.3
+# Script License        : Apache License, Version 2 or later
+# Passing Arguments     : Passing Arguments: 1.Version of package,
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Arumugam N S<asellappen@yahoo.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer            : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=main
+else
+  export VERSION=$1
+fi
+
+if [ -d "curr" ] ; then
+  rm -rf curr
+fi
+# make sure to install make go under /root
+cd
+sudo dnf install -y git make wget tar gcc
+sudo wget https://golang.org/dl/go1.16.7.linux-ppc64le.tar.gz
+sudo tar -xvf go1.16.7.linux-ppc64le.tar.gz
+export PATH=/root/go/bin:$PATH
+export GOPATH=/root/go/bin
```

Comment:
We should not be hardcoding the path like this ? Also, we should be using non-root user as per the header comment

Reply:
correct,taken care